src/tests/link_template_test.cc

```diff
@@ -0,0 +1,67 @@
+#include <cstdlib>
+#include "gtest/gtest.h"
+
+#include "QueryNode.h"
+#include "LinkTemplate.h"
+#include "AtomDBSingleton.h"
+#include "test_utils.h"
+
+using namespace query_engine;
+using namespace query_element;
+
+TEST(LinkTemplate, basics) {
+    
+    setenv("DAS_REDIS_HOSTNAME", "ninjato", 1);
```

Comment:
Tests using this host are failing, no problem for now, we can rebuild the test environment after the PR is merged